### PR TITLE
peg the dashboard to dart 1.16.0

### PR DESCRIPTION
**Pull request by devoncarew**
Experimenting with pegging the dashboard to dart 1.16.0

### CODE REVIEW

**Comment by devoncarew** [comment]
I'm seeing some hard to track down issues compiling the frontend to JS with bleeding edge SDKs, and want to peg the dashboard to 1.16.0 stable for now.

@yjbanov, @sethladd

**Comment by sethladd** [comment]
lgtm

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,11 @@
 language: dart
```
Comment (yjbanov) [inline .travis.yml]: There seems to be a formal method of picking a Dart SDK version: https://docs.travis-ci.com/user/languages/dart#Choosing-Dart-versions-to-test-against

Comment (yjbanov) [inline .travis.yml]: But this is fine too, if it works.

Comment (devoncarew) [inline .travis.yml]: ah, awesome. That seems cleaner :)

**Pull request merged**